### PR TITLE
[17.0][MIG] hr_study: Migration to version 17.0

### PR DESCRIPTION
**Pull request by juanjosesegui-tecnativa**
@Tecnativa TT50076

### CODE REVIEW

**Comment by juanjosesegui-tecnativa** [comment]
ping @victoralmau

File: hr_study/security/security.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="rule_multi_company_hr_study" model="ir.rule">
+        <field name="name">Hr Study multi-company</field>
+        <field name="model_id" ref="model_hr_study" />
+        <field
+            name="domain_force"
+        >['|',('company_id','=',False),('company_id','in',company_ids)]</field>
```
Comment (victoralmau) [inline hr_study/security/security.xml]:
```suggestion
        <field name="domain_force">[('company_id', 'in', company_ids + [False])]</field>
```

File: hr_study/tests/test_hr_study.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            "mail_create_nosubscribe": True,
+            "mail_notrack": True,
+            "no_reset_password": True,
+        }
+        new_test_user(
+            cls.env,
+            login="test-hr_user",
+            groups="hr.group_hr_user",
+            context=ctx,
+        )
```
Comment (victoralmau) [inline hr_study/tests/test_hr_study.py]:
```suggestion
from odoo.tests import Form, new_test_user
from odoo.tests.common import users
from odoo.addons.base.tests.common import BaseCommon


class TestHrStudy(BaseCommon):
    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        cls.study = cls.env["hr.study"].create({"name": "Test study"})
        cls.employee = cls.env["hr.employee"].create({"name": "Test employee"})
        new_test_user(
            cls.env,
            login="test-hr_user",
            groups="hr.group_hr_user",
        )
```

**Review by victoralmau — CHANGES_REQUESTED** [review]

**Review by victoralmau — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**